From d8054f1efa0b97b291a6f8c7fff9090b0fd4328e Mon Sep 17 00:00:00 2001
From: Fabian Vogt <fvogt@suse.de>
Date: Fri, 3 Mar 2023 11:39:58 +0100
Subject: [PATCH 100/104] Add some missing cstdint inclusions (#872)

GCC 13 removed some indirect inclusions of cstdint which this relied on.
Include it explicitly to fix build errors.
---
 include/mbgl/util/geometry.hpp | 2 ++
 include/mbgl/util/string.hpp   | 1 +
 src/mbgl/gl/stencil_mode.hpp   | 2 ++
 3 files changed, 5 insertions(+)

diff --git a/src/3rdparty/mapbox-gl-native/include/mbgl/util/geometry.hpp b/src/3rdparty/mapbox-gl-native/include/mbgl/util/geometry.hpp
index a28c59a47..283ec7dbb 100644
--- a/src/3rdparty/mapbox-gl-native/include/mbgl/util/geometry.hpp
+++ b/src/3rdparty/mapbox-gl-native/include/mbgl/util/geometry.hpp
@@ -1,5 +1,7 @@
 #pragma once
 
+#include <cstdint>
+
 #include <mapbox/geometry/geometry.hpp>
 #include <mapbox/geometry/point_arithmetic.hpp>
 #include <mapbox/geometry/for_each_point.hpp>
diff --git a/src/3rdparty/mapbox-gl-native/include/mbgl/util/string.hpp b/src/3rdparty/mapbox-gl-native/include/mbgl/util/string.hpp
index 13498ccb9..2f6acb641 100644
--- a/src/3rdparty/mapbox-gl-native/include/mbgl/util/string.hpp
+++ b/src/3rdparty/mapbox-gl-native/include/mbgl/util/string.hpp
@@ -3,6 +3,7 @@
 #include <sstream>
 #include <string>
 #include <cassert>
+#include <cstdint>
 #include <cstdlib>
 #include <exception>
 
diff --git a/src/3rdparty/mapbox-gl-native/src/mbgl/gl/stencil_mode.hpp b/src/3rdparty/mapbox-gl-native/src/mbgl/gl/stencil_mode.hpp
index bc959c9a7..fd00370f8 100644
--- a/src/3rdparty/mapbox-gl-native/src/mbgl/gl/stencil_mode.hpp
+++ b/src/3rdparty/mapbox-gl-native/src/mbgl/gl/stencil_mode.hpp
@@ -1,5 +1,7 @@
 #pragma once
 
+#include <cstdint>
+
 #include <mbgl/util/variant.hpp>
 
 namespace mbgl {
-- 
2.47.1

